apis/rds/v1alpha1/custom.go

```diff
@@ -0,0 +1,26 @@
+package v1alpha1
+
+import "github.com/crossplane-contrib/terrajet/pkg/terraform/resource"
+
+// DBClusterExternalNamer is config for external name mechanism of DBCluster.
+var (
+	DBClusterExternalNamer = resource.ExternalNameConfiguration{
```

Comment:
nit (assuming this part will be refactored): 
We also have the `ExternalNamer` naming here which we changed not to cause the confusion mentioned in [this comment](https://github.com/crossplane-contrib/terrajet/pull/66#discussion_r712212312).